APPS-730 Clean Up Payment Method API (#551)
spryker/spryker-docs: docs/scos/user/technology-partners/202108.0/payment-partners/bs-payone/bs-payone.md

 };
```

> The project (demoshop) level `\Pyz\Zed\Checkout\CheckoutDependencyProvider` method `provideBusinessLayerDependencies` container has to be extended with the checkout plugins of Payone. It can be done by adding plugins into related plugins stacks in these methods in `\Pyz\Zed\Checkout\CheckoutDependencyProvider`:

Reviewer: ```suggestion
The project (Demo Shop) level `\Pyz\Zed\Checkout\CheckoutDependencyProvider` method `provideBusinessLayerDependencies` container has to be extended with the checkout plugins of Payone. This can be done by adding plugins to the related plugins stacks in these methods in `\Pyz\Zed\Checkout\CheckoutDependencyProvider`:
```

* `getCheckoutPreConditions`
* `getCheckoutOrderSavers`
* `getCheckoutPostHooks`

> Add the plugins into `\Pyz\Zed\Checkout\CheckoutDependencyProvider`:

Reviewer: ```suggestion
Add the plugins to `\Pyz\Zed\Checkout\CheckoutDependencyProvider`:
```
